src/rest_api/classes/molecule/widgets/overview.clj

```diff
@@ -77,18 +77,9 @@
    :description "Reported uses/affects of the molecule with regards to nematode species biology"})
 
 (defn- convert-value [value]
-  (cond
-    (= (str/lower-case value) "nmr")
-    "NMR"
-
-    (= (str/lower-case value) "lc ms")
-    "LC-MS"
-
-    (= (str/lower-case value) "etoh")
-    "EtOH"
-
-    :else
-    (obj/humanize-ident value)))
+  ;; It appears that this is no longer needed accroding to:
```

Comment:
maybe it was a fancy thing that I added in. Please remove these the comments before merging